tests/model/test_push_item_content.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pytest import raises
+
+from io import UnsupportedOperation, BufferedReader
+
+from pushsource import PushItem
+
+ITEM_SRC = os.path.join(os.path.dirname(__file__), "data/test_file.txt")
+
+
+def _opener(item):
+    # BuffredReader attributes can't be modified.
+    # So override name in subclass to return None
+    class NoNameBR(BufferedReader):
+        @property
+        def name(self):
+            return None
+
+    return NoNameBR(open(item.src, "rb"))
+
+
+def test_push_item_content():
+    """Read pushitem content from the source"""
+    item = PushItem(name="test", src=ITEM_SRC)
+
+    assert item.content().name == ITEM_SRC
```

Comment:
I would strongly suggest that at least one test is using `content` via a `with` statement. This is the most likely way the API will actually be used in practice, and if not tested there is a possibility we've messed it up somehow.

e.g. this test could look more like:

```
with item.content() as f:
  assert f.name == ITEM_SRC
  assert ...
```

Reply:
Done